src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -367,6 +367,12 @@ def conn_wrapper(self, fs_handle, **kwargs):
             return result
         return conn_wrapper
 
+    def nametojson(self, names):
```

Comment:
Please add a doc string saying what this method does.

Reply:
Already added. Please check the latest code.